docs/ai-coder/github-to-tasks.md

```diff
@@ -0,0 +1,164 @@
+- Your Coder Server is accessible from GitHub
+- You have an AI-enabled Task Template that can successfully create a Coder Task. If you don't have a Task Template available, follow our [Getting Started with Tasks Guide](https://coder.com/docs/ai-coder/tasks#getting-started-with-tasks)
+- Check the [Requirements section of the GHA](https://github.com/coder/create-task-action?tab=readme-ov-file#requirements) for specific version requirements for your Coder deployment
+
+This guide can be followed for other usecases beyond bugs like updating documetantion or implementing a small feature, but may require minor changes to file names and the prompts provided to the Coder Task.
+
+### Step 1: Create a GitHub Workflow file
+
+In your repository, create a new file in the `./.github/workflows/` directory named `triage-bug.yaml`. Within that file, add the following code: 
+
+```yaml
+name: Start Coder Task
+
+on:
+  issues:
+    types:
+      - labeled
+
+permissions:
+  issues: write
+
+jobs:
+  start-coder-task:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'coder'
+    steps:
+      - name: Start Coder Task
+        uses: coder/start-coder-task@v0.0.2
+        with:
+          coder-url: ${{ secrets.CODER_URL }}
+          coder-token: ${{ secrets.CODER_TOKEN }}
+          coder-organization: "default"
+          coder-template-name: "my-template"
+          coder-task-name-prefix: "gh-task"
+          coder-task-prompt: "Use the gh CLI to read ${{ github.event.issue.html_url }}, write an appropriate plan for solving the issue to PLAN.md, and then wait for feedback."
+          github-user-id: ${{ github.event.sender.id }}
+          github-issue-url: ${{ github.event.issue.html_url }}
+          github-token: ${{ github.token }}
+          comment-on-issue: true
+```
+
+This code will perform the following actions:
+
+- Create a Coder Task when you apply the `coder` label to an existing GitHub issue
+- Pass as a prompt to the Coder Task
+    
+    1. Use the GitHub CLI to access and read the content of the linked GitHub issue
+    1. Generate an initial implementation plan to solve the bug
+    1. Write that plan to a `PLAN.md` file
+    1. Wait for additional input
+
+The prompt text can be modified to not wait for additional human input, but continue with implementing the proposed solution and creating a PR for example.
+
+### Step 2: Setup the Required Secrets & Inputs
+
+The GHA has multiple required inputs that require configuring before the workflow can successfully operate. 
+
+You must set the following inputs as secrets within your repository:
+
+- `coder-url`: the URL of your Coder deployment, e.g. https://coder.example.com
+- `coder-token`: follow our [API Tokens documentation](https://coder.com/docs/admin/users/sessions-tokens#long-lived-tokens-api-tokens) to generate a token
```

Comment:
We should also mention this requires an admin/org-level token with the following permissions:

- Read users in organization
- Create tasks for any user

(Also mentioned in GHA readme.)